test/unittest/unittest_nntrainer_modelfile.cpp

```diff
@@ -648,10 +655,10 @@ TEST(nntrainerIniTest, backbone_p_20) {
  * it should be referred relative to the .ini
  *
  */
-TEST(nntrainerIniTest, backbone_relative_to_ini_p) {
-  ScopedIni b{getResPath("base"), {nw_base, batch_normal}};
+TEST(nntrainerIniTest, DISABLED_backbone_relative_to_ini_p) {
+  ScopedIni b{getResPath("base"), {nw_base_cross, batch_normal}};
   ScopedIni s{getResPath("original"),
-              {nw_base + "loss=mse", adam, input,
+              {nw_base_cross + "loss=mse", adam, input,
```

Comment:
I think this will override nw_base to mse.

Reply:
this maintains the original code implemented (nw_base(old) == nw_base_cross)
And yes, hopefully this will override to mse.